src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx

```diff
@@ -95,4 +103,85 @@ describe('Testing AddOnEntry', () => {
     expect(getByText('Test addon description')).toBeInTheDocument();
     expect(getByText('Test User')).toBeInTheDocument();
   });
+  it('Uninstall Button works correctly', async () => {
+    const props = {
+      id: '1',
+      title: 'Test Addon',
+      description: 'Test addon description',
+      createdBy: 'Test User',
+      component: 'string',
+      installed: true,
+      configurable: true,
+      modified: true,
+      isInstalled: true,
+      uninstalledOrgs: [],
+      enabled: true,
+      getInstalledPlugins: (): { sample: string } => {
+        return { sample: 'sample' };
+      },
+    };
+
+    const { findByText, getByTestId } = render(
+      <MockedProvider addTypename={false} link={link}>
+        <Provider store={store}>
+          <BrowserRouter>
+            <I18nextProvider i18n={i18nForTest}>
+              <ToastContainer />
+              {<AddOnEntry {...props} />}
+            </I18nextProvider>
+          </BrowserRouter>
+        </Provider>
+      </MockedProvider>
+    );
+    await wait(100);
+    const btn = getByTestId('AddOnEntry_btn_install');
+    await userEvent.click(btn);
+    await wait(100);
+    expect(btn.innerHTML).toMatch(/Install/i);
+    expect(
+      await findByText('This feature is now removed from your organization')
+    ).toBeInTheDocument();
+    await userEvent.click(btn);
+    await wait(100);
+
+    expect(btn.innerHTML).toMatch(/Uninstall/i);
+    expect(
+      await findByText('This feature is now enabled in your organization')
+    ).toBeInTheDocument();
+  });
+
+  it('Check if uninstalled orgs includes current org', async () => {
+    const props = {
+      id: '1',
+      title: 'Test Addon',
+      description: 'Test addon description',
+      createdBy: 'Test User',
+      component: 'string',
+      installed: true,
+      configurable: true,
+      modified: true,
+      isInstalled: true,
+      uninstalledOrgs: ['undefined'],
+      enabled: true,
+      getInstalledPlugins: (): { sample: string } => {
+        return { sample: 'sample' };
+      },
+    };
+
+    const { getByTestId } = render(
+      <MockedProvider addTypename={false} link={link}>
+        <Provider store={store}>
+          <BrowserRouter>
+            <I18nextProvider i18n={i18nForTest}>
+              {<AddOnEntry {...props} />}
+            </I18nextProvider>
+          </BrowserRouter>
+        </Provider>
+      </MockedProvider>
+    );
+    await wait(100);
+    const btn = getByTestId('AddOnEntry_btn_install');
+    expect(btn.innerHTML).toMatch(/install/i);
+    debug();
```

Comment:
Why is `debug()` used here?

Reply:
I used the `debug()` function to debug the test case using jest-preview but forgot to remove it, thank you for pointing this out. I'm removing it now.